<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>官文亮</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">SerialTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/serialtool.qrc">
    <normaloff>:/SerialTool/images/logo.ico</normaloff>:/SerialTool/images/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOption"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="portSwitchAction"/>
    <addaction name="portRunAction"/>
    <addaction name="clearAction"/>
    <addaction name="separator"/>
    <addaction name="portSetAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionVisibleToolbar"/>
    <addaction name="actionVisibleStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionStaysOnTop"/>
    <addaction name="actionRemoveConfig"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWiki"/>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="toolBar1">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="portSetAction"/>
   <addaction name="separator"/>
   <addaction name="portSwitchAction"/>
   <addaction name="portSwitchAction2"/>
   <addaction name="portRunAction"/>
   <addaction name="clearAction"/>
   <addaction name="separator"/>
   <addaction name="actionOption"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/save.ico</normaloff>:/SerialTool/images/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/config.ico</normaloff>:/SerialTool/images/config.ico</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="portSwitchAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/connect.ico</normaloff>:/SerialTool/images/connect.ico</iconset>
   </property>
   <property name="text">
    <string>Open Port</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="portSwitchAction2">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/start.ico</normaloff>
     <normalon>:/SerialTool/images/exit.ico</normalon>:/SerialTool/images/start.ico</iconset>
   </property>
   <property name="text">
    <string>Auto Restart Port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="portRunAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/pause.ico</normaloff>
     <activeoff>:/SerialTool/images/pause.png</activeoff>:/SerialTool/images/pause.ico</iconset>
   </property>
   <property name="text">
    <string>Pause Tx/Rx</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/clear.ico</normaloff>:/SerialTool/images/clear.ico</iconset>
   </property>
   <property name="text">
    <string>Clear Buffer</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="portSetAction">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/port config.ico</normaloff>:/SerialTool/images/port config.ico</iconset>
   </property>
   <property name="text">
    <string>Port Settings</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/exit.ico</normaloff>:/SerialTool/images/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionVisibleToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tool Bar</string>
   </property>
  </action>
  <action name="actionVisibleStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="text">
    <string>Wiki</string>
   </property>
   <property name="toolTip">
    <string>Wiki</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource/serialtool.qrc">
     <normaloff>:/SerialTool/images/open.ico</normaloff>:/SerialTool/images/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStaysOnTop">
   <property name="icon">
    <iconset>
     <normaloff>:/SerialTool/images/pin_up.png</normaloff>:/SerialTool/images/pin_up.png</iconset>
   </property>
   <property name="text">
    <string>Stays On Top</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionRemoveConfig">
   <property name="text">
    <string>Reset Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/serialtool.qrc"/>
 </resources>
 <connections/>
</ui>
